spec/features/organizer_ability_spec.rb

```diff
@@ -108,16 +108,34 @@
       visit edit_admin_conference_program_path(conference.short_title)
       expect(current_path).to eq(edit_admin_conference_program_path(conference.short_title))
 
+      # only event type exits
```

Comment:
You should just keep the ones that are helpful to our code, ie I can tell you are calling the new action from the path, you don't have to put that in a comment on the previous line. However I found the comment about which type exists useful, as you can't tell in a glance which one exists, and which not.